omnibus/config/projects/chef.rb

```diff
@@ -51,15 +51,27 @@
 # Chef Release version pinning
 override :chef, version: "local_source"
 override :ohai, version: "master"
+override :"rb-readline", version: "v0.5.3"
```

Comment:
Is this override needed? Shouldn't just make the `default_version` of the `rb-readline` software def `v0.5.3`.